persia/data.py

```diff
@@ -77,13 +78,14 @@ class PersiaChannel(IterableChannelBase):
 
     Arguments:
         buffer_size (int): PyPersiaBatchDataChannel buffer size
-        replica_info (PyPersiaReplicaInfo): replica info of current process to enable the data reorder ability
+        replica_info (PyPersiaReplicaInfo, optional): replica info of current process to enable the data reorder ability
+        async_iterator (bool, optional): launch the thread to generate the data asynchronous
     """
 
     def __init__(
         self,
         buffer_size: int,
-        replica_info: PyPersiaReplicaInfo = None,
+        replica_info: Optional[PyPersiaReplicaInfo] = None,
```

Comment:
Better to move all optional args behind to the required?

Reply:
`Optional` means that it can set to `None` as default value to pass the pytype test